### PR TITLE
Post-release mobile sdk doc edits

### PR DESCRIPTION
**Pull request by ghost**
Jira: 146904

### CODE REVIEW

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx [hunk 1]
```diff
@@ -1633,7 +1637,130 @@ Here's a sample configuration:
...
+        <TabsPageItem id="maui">
+
+The .NET MAUI agent SDK allows you to configure default settings to change the behavior of the agent. 
+
+If used, make sure to add the configuration flags just before calling `appToken`. 
+
+Here's a sample configuration:
+
+```csharp
+```
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx:1649]: @ndesai-newrelic Can you provide a .NET MAUI sample configuration (similar to the other hybrid agents)?

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx [hunk 2]
```diff
@@ -1472,6 +1476,10 @@ If used, make sure to add the configuration flags just before calling `appToken`
 
 Here's a sample configuration:
 
+```typescript
+
+```
+
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx:1482]:
```suggestion
### Objective-c

```objectivec

```

### Swift

```swift

```

```

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx [hunk 3]
```diff
@@ -1472,6 +1476,10 @@ If used, make sure to add the configuration flags just before calling `appToken`
 
 Here's a sample configuration:
 
+```typescript
+
+```
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx:1481]: @ndesai-newrelic Same here, can you provide a Capacitor config example?

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx [hunk 4]
```diff
@@ -2041,131 +2172,7 @@ If used, make sure to add the configuration flags just before calling `appToken`
 Here's a sample configuration:
 
 ```csharp
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx:2174]: @ndesai-newrelic Same here, can you provide a Xamarin sample config?

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx (hunk 3 above)
Comment (ndesai-newrelic) [reply src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx:1139]: it is already on the website.

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx (hunk 4 above)
Comment (ndesai-newrelic) [reply src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx]: public App ()
            {
                InitializeComponent();

                MainPage = new MainPage();
                Application.Current.PageAppearing += OnPageAppearing;
                Application.Current.PageDisappearing += PageDisappearing;

                CrossNewRelicClient.Current.HandleUncaughtException();
                CrossNewRelicClient.Current.TrackShellNavigatedEvents()
      // Set optional agent configuration
      // Options are: crashReportingEnabled, loggingEnabled, logLevel, collectorAddress, crashCollectorAddress
      AgentStartConfiguration agentConfig = new AgentStartConfiguration(true, true, LogLevel.INFO, "mobile-collector.newrelic.com", "mobile-crash.newrelic.com");
      if (Device.RuntimePlatform == Device.Android)
                {
                    //CrossNewRelicClient.Current.Start("<APP-TOKEN-HERE>");
                    // Start with optional agent configuration
                    CrossNewRelicClient.Current.Start("<APP-TOKEN-HERE", agentConfig);
                }
                else if (Device.RuntimePlatform == Device.iOS)
                {
                    //CrossNewRelicClient.Current.Start("<APP-TOKEN-HERE>");
                    // Start with optional agent configuration 
                    CrossNewRelicClient.Current.Start("<APP-TOKEN-HERE", agentConfig);
                }
            }
        }

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx (hunk 1 above)
Comment (ndesai-newrelic) [reply src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx:1311]:
```suggestion
Here's a sample configuration:

```csharp
 using NewRelic.MAUI.Plugin;
            ...
    public App()
        {
            InitializeComponent();

            MainPage = new AppShell();

            CrossNewRelic.Current.HandleUncaughtException();
            CrossNewRelic.Current.TrackShellNavigatedEvents();

            // Set optional agent configuration
            // Options are: crashReportingEnabled, loggingEnabled, logLevel, collectorAddress, crashCollectorAddress
            AgentStartConfiguration agentConfig = new AgentStartConfiguration(true, true, LogLevel.INFO, "mobile-collector.newrelic.com", "mobile-crash.newrelic.com");

            if (DeviceInfo.Current.Platform == DevicePlatform.Android)
            {
                // CrossNewRelic.Current.Start("<APP-TOKEN-HERE>");
                // Start with optional agent configuration 
                 CrossNewRelic.Current.Start("<APP-TOKEN-HERE", agentConfig);
            }
            else if (DeviceInfo.Current.Platform == DevicePlatform.iOS)
            {
                // CrossNewRelic.Current.Start("<APP-TOKEN-HERE>");
                // Start with optional agent configuration 
                 CrossNewRelic.Current.Start("<APP-TOKEN-HERE", agentConfig);
            }
        }



    }
```

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx (hunk 4 above)
Comment (ndesai-newrelic) [inline src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx]: public App ()
            {
                InitializeComponent();

                MainPage = new MainPage();
                Application.Current.PageAppearing += OnPageAppearing;
                Application.Current.PageDisappearing += PageDisappearing;

                CrossNewRelicClient.Current.HandleUncaughtException();
                CrossNewRelicClient.Current.TrackShellNavigatedEvents()
      // Set optional agent configuration
      // Options are: crashReportingEnabled, loggingEnabled, logLevel, collectorAddress, crashCollectorAddress
      AgentStartConfiguration agentConfig = new AgentStartConfiguration(true, true, LogLevel.INFO, "mobile-collector.newrelic.com", "mobile-crash.newrelic.com");
      if (Device.RuntimePlatform == Device.Android)
                {
                    //CrossNewRelicClient.Current.Start("<APP-TOKEN-HERE>");
                    // Start with optional agent configuration
                    CrossNewRelicClient.Current.Start("<APP-TOKEN-HERE", agentConfig);
                }
                else if (Device.RuntimePlatform == Device.iOS)
                {
                    //CrossNewRelicClient.Current.Start("<APP-TOKEN-HERE>");
                    // Start with optional agent configuration 
                    CrossNewRelicClient.Current.Start("<APP-TOKEN-HERE", agentConfig);
                }
            }
        }

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/mobile-sdk-api-guide.mdx [hunk 5]
```diff
@@ -16,21 +16,14 @@ While our mobile monitoring solution offers powerful out-of-the-box data collect
...
-    * [Android SDK usage](#ios-sdk-usage)
-    * [iOS SDK usage](#android-sdk-usage)
-
-3. Select a customization from the table below and follow the method's documentation.
+2. Select a customization from the table below and follow the method's documentation.
 
 ## Available customizations [#methods]
 
-The table below lists some supported SDK methods to customize your native or hybrid agent instrumentation.
+The table below lists supported SDK methods to customize your mobile agent instrumentation. Click on a method's documentation to view syntax and usage examples for each agent type.
```
Comment (nwhite222) [inline src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/mobile-sdk-api-guide.mdx:26]:
```suggestion
The table below lists supported SDK methods to customize your mobile agent instrumentation. Select a method to view syntax and usage examples for each agent type.
```

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb.mdx [hunk 6]
```diff
@@ -64,9 +67,7 @@ NewRelic.recordBreadcrumb(
...
 
-In addition to whatever custom attributes you choose, the event will also have associated [session attributes](/docs/insights/explore-data/attributes/mobile-default-attributes-insights#mobile-list). Unlike with using [`setAttribute`](/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/set-attribute), adding attributes to a breadcrumb event adds them only to that event; they are not session attributes.
-
-For related Android SDK API calls, see [Send custom attributes and events](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/guide-custom-instrumentation-api#send-insights).
+In addition to whatever custom attributes you choose, the event will also have associated [session attributes](/docs/insights/explore-data/attributes/mobile-default-attributes-insights#mobile-list). Unlike with using [`setAttribute`](docs/mobile-monitoring/new-relic-mobile/mobile-sdk/create-attribute/), adding attributes to a breadcrumb event adds them only to that event; they are not session attributes.
```
Comment (nwhite222) [inline src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb.mdx:70]:
```suggestion
In addition to any custom attributes you choose, the event will also have associated [session attributes](/docs/insights/explore-data/attributes/mobile-default-attributes-insights#mobile-list). Unlike with using [`setAttribute`](docs/mobile-monitoring/new-relic-mobile/mobile-sdk/create-attribute/), adding attributes to a breadcrumb event adds them only to that event; they are not session attributes.
```

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb.mdx [hunk 7]
```diff
@@ -180,9 +181,9 @@ NewRelic.recordBreadcrumb(string $Name, map $eventAttributes)
 
 This call creates and records a `MobileBreadcrumb` event, which can be queried with NRQL and in the [crash event trail](/docs/mobile-monitoring/mobile-monitoring-ui/crashes/mobile-crash-event-trail). Mobile breadcrumbs are useful for crash analysis; create them for app activity that you think will help you troubleshoot crashes.
 
-In addition to whatever custom attributes you choose, the event will also have associated [session attributes](/docs/insights/explore-data/attributes/mobile-default-attributes-insights#mobile-list). Unlike using [`setAttribute`](/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/set-attribute), adding attributes to a breadcrumb event adds them only to that event; they are not session attributes.
+In addition to whatever custom attributes you choose, the event will also have associated [session attributes](/docs/insights/explore-data/attributes/mobile-default-attributes-insights#mobile-list). Unlike using [`setAttribute`](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/create-attribute/), adding attributes to a breadcrumb event adds them only to that event; they are not session attributes.
```
Comment (nwhite222) [inline src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb.mdx:184]:
```suggestion
In addition to any custom attributes you choose, the event will also have associated [session attributes](/docs/insights/explore-data/attributes/mobile-default-attributes-insights#mobile-list). Unlike using [`setAttribute`](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/create-attribute/), adding attributes to a breadcrumb event adds them only to that event; they are not session attributes.
```

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb.mdx [hunk 8]
```diff
@@ -268,9 +269,7 @@ recordBreadcrumb(options: { name: string; eventAttributes: object; }) => void
...
 
-In addition to whatever custom attributes you choose, the event will also have associated [session attributes](/docs/insights/explore-data/attributes/mobile-default-attributes-insights#mobile-list). Unlike with using [`setAttribute`](/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/set-attribute), adding attributes to a breadcrumb event adds them only to that event; they are not session attributes.
-
-For related Android SDK API calls, see [Send custom attributes and events](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/guide-custom-instrumentation-api#send-insights).
+In addition to whatever custom attributes you choose, the event will also have associated [session attributes](/docs/insights/explore-data/attributes/mobile-default-attributes-insights#mobile-list). Unlike with using [`setAttribute`](docs/mobile-monitoring/new-relic-mobile/mobile-sdk/create-attribute/), adding attributes to a breadcrumb event adds them only to that event; they are not session attributes.
```
Comment (nwhite222) [inline src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb.mdx:272]:
```suggestion
In addition to any custom attributes you choose, the event will also have associated [session attributes](/docs/insights/explore-data/attributes/mobile-default-attributes-insights#mobile-list). Unlike with using [`setAttribute`](docs/mobile-monitoring/new-relic-mobile/mobile-sdk/create-attribute/), adding attributes to a breadcrumb event adds them only to that event; they are not session attributes.
```

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb.mdx [hunk 9]
```diff
@@ -345,9 +344,8 @@ recordBreadcrumb(eventName: string, attributes?: {[key: string]: boolean | numbe
 
 This call creates and records a `MobileBreadcrumb` event, which can be queried with NRQL and in the [crash event trail](/docs/mobile-monitoring/mobile-monitoring-ui/crashes/mobile-crash-event-trail). Mobile breadcrumbs are useful for crash analysis; they should be created for app activity that may be helpful for troubleshooting crashes.
 
-In addition to whatever custom attributes you choose, the event will also have associated [session attributes](/docs/insights/explore-data/attributes/mobile-default-attributes-insights#mobile-list). Unlike with using [`setAttribute`](/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/set-attribute), adding attributes to a breadcrumb event adds them only to that event; they are not session attributes.
+In addition to whatever custom attributes you choose, the event will also have associated [session attributes](/docs/insights/explore-data/attributes/mobile-default-attributes-insights#mobile-list). Unlike with using [`setAttribute`](docs/mobile-monitoring/new-relic-mobile/mobile-sdk/create-attribute/), adding attributes to a breadcrumb event adds them only to that event; they are not session attributes.
```
Comment (nwhite222) [inline src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb.mdx:347]:
```suggestion
In addition to any custom attributes you choose, the event will also have associated [session attributes](/docs/insights/explore-data/attributes/mobile-default-attributes-insights#mobile-list). Unlike with using [`setAttribute`](docs/mobile-monitoring/new-relic-mobile/mobile-sdk/create-attribute/), adding attributes to a breadcrumb event adds them only to that event; they are not session attributes.
```

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb.mdx [hunk 10]
```diff
@@ -411,21 +409,19 @@ Returns `true` if the event is recorded successfully, or `false` if not.
...
 
-In addition to whatever custom attributes you choose, the event will also have associated [session attributes](/docs/insights/explore-data/attributes/mobile-default-attributes-insights#mobile-list). Unlike with using [`setAttribute`](/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/set-attribute), adding attributes to a breadcrumb event adds them only to that event; they are not session attributes.
-
-For related Android SDK API calls, see [Send custom attributes and events](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/guide-custom-instrumentation-api#send-insights).
+In addition to whatever custom attributes you choose, the event will also have associated [session attributes](/docs/insights/explore-data/attributes/mobile-default-attributes-insights#mobile-list). Unlike with using [`setAttribute`](docs/mobile-monitoring/new-relic-mobile/mobile-sdk/create-attribute/), adding attributes to a breadcrumb event adds them only to that event; they are not session attributes.
```
Comment (nwhite222) [inline src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb.mdx:424]:
```suggestion
In addition to any custom attributes you choose, the event will also have associated [session attributes](/docs/insights/explore-data/attributes/mobile-default-attributes-insights#mobile-list). Unlike with using [`setAttribute`](docs/mobile-monitoring/new-relic-mobile/mobile-sdk/create-attribute/), adding attributes to a breadcrumb event adds them only to that event; they are not session attributes.
```

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb.mdx [hunk 11]
```diff
@@ -479,25 +477,30 @@ Returns `true` if the event is recorded successfully, or `false` if not.
...
```
Comment (nwhite222) [inline src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb.mdx:503]:
```suggestion
In addition to any custom attributes you choose, the event will also have associated [session attributes](/docs/insights/explore-data/attributes/mobile-default-attributes-insights#mobile-list). Unlike with using [`setAttribute`](docs/mobile-monitoring/new-relic-mobile/mobile-sdk/create-attribute/), adding attributes to a breadcrumb event adds them only to that event; they are not session attributes.
```

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb.mdx [hunk 12]
```diff
@@ -553,26 +554,23 @@ Returns `true` if the event is recorded successfully, or `false` if not.
...
```
Comment (nwhite222) [inline src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb.mdx:573]:
```suggestion
In addition to any custom attributes you choose, the event will also have associated [session attributes](/docs/insights/explore-data/attributes/mobile-default-attributes-insights#mobile-list). Unlike with using [`setAttribute`](docs/mobile-monitoring/new-relic-mobile/mobile-sdk/create-attribute/), adding attributes to a breadcrumb event adds them only to that event; they are not session attributes.
```

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb.mdx [hunk 13]
```diff
@@ -651,9 +643,7 @@ RecordBreadcrumb(string name, Dictionary<string, object> attributes): bool;
...
```
Comment (nwhite222) [inline src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb.mdx:646]:
```suggestion
In addition to any custom attributes you choose, the event will also have associated [session attributes](/docs/insights/explore-data/attributes/mobile-default-attributes-insights#mobile-list). Unlike with using [`setAttribute`](docs/mobile-monitoring/new-relic-mobile/mobile-sdk/create-attribute/), adding attributes to a breadcrumb event adds them only to that event; they are not session attributes.
```

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-custom-events.mdx [hunk 14]
```diff
@@ -520,17 +520,19 @@ Returns `true` if the event is recorded successfully, or `false` if not.
...
+```csharp
+RecordCustomEvent(string eventType, string eventName, Dictionary<string, object> attributes): bool;
 ```
 
 ## Description [#description]
 
-Creates and records a custom event for use in New Relic dashboards.
+Creates and records a [custom event](/docs/insights/new-relic-insights/adding-querying-data/custom-attributes-events-new-relic-mobile#What-are-events-in-New-Relic-Mobile), for use in NRQL.
+
+In addition to whatever custom attributes you choose, the event will also have associated [session attributes](/docs/insights/explore-data/attributes/mobile-default-attributes-insights#mobile-list). Unlike with using [`setAttribute`](docs/mobile-monitoring/new-relic-mobile/mobile-sdk/create-attribute/), adding attributes to a breadcrumb event adds them only to that event; they are not session attributes.
```
Comment (nwhite222) [inline src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-custom-events.mdx:535]:
```suggestion
In addition to any custom attributes you choose, the event will also have associated [session attributes](/docs/insights/explore-data/attributes/mobile-default-attributes-insights#mobile-list). Unlike with using [`setAttribute`](docs/mobile-monitoring/new-relic-mobile/mobile-sdk/create-attribute/), adding attributes to a breadcrumb event adds them only to that event; they are not session attributes.
```

File: src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/mobile-crash-event-trail.mdx [hunk 15]
```diff
@@ -28,9 +28,9 @@ When a mobile app crashes and you don't know why, you can study what happened ri
...
 </figcaption>
 
-The crash event trail shows all mobile [event types](/docs/insights/explore-add-data/custom-events/insert-query-custom-mobile-app-events-attributes#event-definition) leading up to a crash. You can use the [iOS SDK](/docs/mobile-monitoring/new-relic-mobile-ios/install-configure/work-ios-sdk-api) or [Android SDK](/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/recordbreadcrumb) to create custom `MobileBreadcrumb` events that track whatever app activity you think would help you diagnose a crash.
+The crash event trail shows all mobile [event types](/docs/insights/explore-add-data/custom-events/insert-query-custom-mobile-app-events-attributes#event-definition) leading up to a crash. You can use the [Record breadkcrumb SDK](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb/) to create custom `MobileBreadcrumb` events that track whatever app activity you think would help you diagnose a crash.
```
Comment (nwhite222) [inline src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/mobile-crash-event-trail.mdx:31]:
```suggestion
The crash event trail shows all mobile [event types](/docs/insights/explore-add-data/custom-events/insert-query-custom-mobile-app-events-attributes#event-definition) leading up to a crash. You can use the [Record breadkcrumb SDK](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb/) to create custom `MobileBreadcrumb` events that track app activity you think would help you diagnose a crash.
```

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-errors.mdx [hunk 16]
```diff
@@ -16,11 +16,17 @@ redirects:
...
-        <TabsBarItem id="maui">
-            .NET MAUI
+        <TabsBarItem id="capacitor">
+            Capacitor
+        </TabsBarItem>
+        <TabsBarItem id="flutter">
+            Flutter
+        </TabsBarItem>
+        <TabsBarItem id="react">
+            React native
```
Comment (nwhite222) [inline src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-errors.mdx:29]:
```suggestion
            React Native
```

Comment (nwhite222) [reply src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-errors.mdx:29]: Also, is there any reason you can think of why the 'React Native' tab appears to be so much smaller than the other agent pages?

**Review by nwhite222 — DISMISSED** [review]
Looks great! Such a comprehensive improvement.

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-errors.mdx (hunk 16 above)
Comment (ally-sassman) [reply src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-errors.mdx:29]: I'm not sure, but that's a good catch. I'll look into it!

File: scripts/utils/migrate/instruction-set.js [hunk 17]
```diff
@@ -511,7 +511,7 @@ module.exports = [
       'Android SDK API',
     ],
     node: {
-      path: '/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api',
+      path: '/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/mobile-sdk-api-guide/',
```
Comment (ally-sassman) [inline scripts/utils/migrate/instruction-set.js:514]:
```suggestion
      path: '/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api',
```

File: scripts/utils/migrate/instruction-set.js [hunk 18]
```diff
@@ -523,7 +523,7 @@ module.exports = [
       'iOS SDK API',
     ],
     node: {
-      path: '/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api',
+      path: '/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/mobile-sdk-api-guide/',
```
Comment (ally-sassman) [inline scripts/utils/migrate/instruction-set.js:526]:
```suggestion
      path: '/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api',
```

**Pull request merged**